.circleci/config.yml

```diff
@@ -246,9 +263,106 @@ jobs:
              'until [ `grep "Registered beacons" -s -r /var/log/scion/ | wc -l` -ge 5 ]; do sleep 0.1; done; \
               scmp echo -c 10 -local 20-ffaa:1:4,[127.0.0.1] -remote 20-ffaa:0:1405,[127.0.0.1]'
 
+  upgrade_integration:
+    machine:
+      image: ubuntu-1604:201903-01
+
+    steps:
+      - checkout
```

Comment:
Just for clarity: can you move this checkout to just before the place where the new container is built?
Also, it would be helpful for a quick overview to name the steps clearly "old" or "new", e.g. "Old -- Checkout", "Old -- Build container images", and then "New -- Checkout" etc.